scripts/test-ssr/src/utils/constants.ts

```diff
@@ -1 +1,2 @@
+export const CHROME_VERSION = 100;
```

Comment:
Chrome version is hardcoded now, starting browser to get a version for every package run will be too much

Reply:
Just curious, why'd you pick v100?